app/dashboard/models.py

```diff
@@ -904,8 +904,6 @@ def value_in_usdt_now(self):
         decimals = 1
         if self.tokenName == 'USDT':
             return float(self.amount)
-        if self.tokenName == 'DAI':
```

Comment:
instead of removing these lines can we make them say

`float(self.amount)`

instead?

Reply:
Sure.